node/src/components/contract_runtime/config.rs

```diff
@@ -26,6 +26,11 @@ pub struct Config {
     ///
     /// Defaults to 5.
     max_query_depth: Option<u64>,
+
+    /// Enable synchronizing to disk only after each block is written.
+    ///
+    /// Defaults to true.
+    enable_manual_sync: bool,
```

Comment:
This needs to be changed to an `Option<bool>` if we want to leave it commented out in the config.toml files.

Reply:
Done